sdk-tests/src/test/java/io/dapr/it/spring/messaging/DaprSpringMessagingIT.java

```diff
@@ -65,22 +68,17 @@ public class DaprSpringMessagingIT {
       .withLogConsumer(outputFrame -> System.out.println(outputFrame.getUtf8String()))
       .withAppChannelAddress("host.testcontainers.internal");
 
-  @DynamicPropertySource
-  static void daprProperties(DynamicPropertyRegistry registry) {
-    org.testcontainers.Testcontainers.exposeHostPorts(8080);
-
-    registry.add("dapr.http.endpoint", DAPR_CONTAINER::getHttpEndpoint);
-    registry.add("dapr.grpc.endpoint", DAPR_CONTAINER::getGrpcEndpoint);
-    registry.add("dapr.grpc.port", DAPR_CONTAINER::getGrpcPort);
-    registry.add("dapr.http.port", DAPR_CONTAINER::getHttpPort);
-  }
-
   @Autowired
   private DaprMessagingTemplate<String> messagingTemplate;
 
   @Autowired
   private TestRestController testRestController;
 
+  @BeforeEach
```

Comment:
This should probably be `@BeforeAll` similar to static method we used to have for `@DynamicPropertySource`